File: Reporting/Reporting.Data/Model/ContactInfo.cs
namespace Reporting.Data.Model;

public class ContactInfo
{
    public Guid Id { get; set; }
    
    public Guid HotelId { get; set; }
    
    public string Name { get; set; }
    
    public string Phone { get; set; }
    
    public string Email { get; set; }
}

File: HotelManagementService/src/models/contact_info.go
package models

import (
	"github.com/google/uuid"
)

type ContactInfo struct {
	Id      uuid.UUID `gorm:"primaryKey;type:uuid;default:uuid_generate_v4();column:Id" json:"id"`
	HotelId uuid.UUID `gorm:"type:uuid;not null;column:HotelId" json:"hotel_id"`
	Name    string    `gorm:"column:Name" json:"name"`
	Phone   string    `gorm:"column:Phone" json:"phone"`
	Email   string    `gorm:"column:Email" json:"email"`
}

func (ContactInfo) TableName() string {
	return "ContactInfos"
}
